### PR TITLE
Added model files steps and code

### PR DESCRIPTION
**Pull request by anilsingla**
coe and steps to add model files

### CODE REVIEW

File: playbooks/besman-counterfit-0.0.1-playbook.sh [hunk 1]
```diff
@@ -77,6 +85,9 @@ function __besman_execute() {
     done
 
     [[ -z $COUNTERFIT_ATTACKID ]] && __besman_echo_red "Attack Id is not set. Required. Please set it and try again." && return 1
+    [[ ! -f counterfit/targets/$BESMAN_ARTIFACT_NAME.py ]] && __besman_echo_red "$BESMAN_ARTIFACT_NAME.py not copied to targets folder." && return 1
```
Comment (asa1997) [inline playbooks/besman-counterfit-0.0.1-playbook.sh:88]: Provide the complete path everywhere - $HOME/counterfit/counterfit/targets..

File: playbooks/besman-counterfit-0.0.1-playbook.sh [hunk 2]
```diff
@@ -20,7 +20,17 @@ function __besman_init() {
...
     done
-
+    
+    local status_code=$(curl -o /dev/null -s -w "%{http_code}\n" $BESMAN_ARTIFACT_URL)
+    if [ "$status_code" -ne 200 ]; then
+       __besman_echo_red "The $BESMAN_ARTIFACT_URL is not found."
+       __besman_echo_red "Create the model repository on github/gitlab and try again."
+       __besman_echo_red "Make the the following files available in repository."
+       __besman_echo_red "   1. $BESMAN_ARTIFACT_NAME.h5"
+       __besman_echo_red "   2. counterfit/$BESMAN_ARTIFACT_NAME.npz"
```
Comment (asa1997) [inline playbooks/besman-counterfit-0.0.1-playbook.sh:30]: The repo need not contain a counterfit sub-directory. Just mention the file names.

**Review by asa1997 — CHANGES_REQUESTED** [review]

File: playbooks/besman-counterfit-0.0.1-playbook.sh (hunk 2 above)
Comment (anilsingla) [reply playbooks/besman-counterfit-0.0.1-playbook.sh]: Done

File: playbooks/besman-counterfit-0.0.1-playbook.sh (hunk 1 above)
Comment (anilsingla) [reply playbooks/besman-counterfit-0.0.1-playbook.sh]: Changed

**Review by asa1997 — APPROVED** [review]
@anilsingla LGTM

**Pull request merged**